Content/ForceProcess.md

```diff
@@ -9,11 +9,18 @@ _ForceProcess_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md). It spec
    [DefaultBox]
    ForceProcess=iexplore.exe
    ForceProcess=firefox.exe
+   ForceProcess=App*.exe
+   ForceProcess=App?.exe
    [MailBox]
    ForceProcess=outlook.exe
+   ForceProcess=c?ick??.exe

```

Comment:
```suggestion
   ForceProcess=click??.exe
```

In the lines below, there is a specific `click??` reference.

Reply:
`c?ick??` shows the use of wildcards in the middle